api/openstorage.yaml

```diff
@@ -803,16 +815,26 @@ paths:
                      }
                     }
               ```
-          required: true
-          schema:
-            $ref: '#/definitions/VolumeCreateRequest'
-      tags:
-        - Volumes
-      responses:
-        '200':
-          description: UUID of newly created volume
-          schema:
-            $ref: '#/definitions/VolumeCreateResponse'
+            required: true
+            schema:
+              $ref: '#/definitions/VolumeCreateRequest'
+        tags:
+          - Volumes
+        responses:
+          '200':
+            description: UUID of neewly created volume
+            schema:
+              $ref: '#/definitions/VolumeCreateResponse'
+            examples:
+             application/json: |-
+              {
+                 "cmd": "create",
+                 "status": "Ok",
```

Comment:
There is no error code.
it just returns as below


{
  "volume_response": {
    "error": "Already exists"
  }
}